gtk4/sample/examples/application5/exampleapp.rb

```diff
@@ -23,6 +23,12 @@
 # * https://gitlab.gnome.org/GNOME/gtk/-/blob/main/examples/application5/gears-menu.ui
 # License: LGPL2.1-or-later
 
+if File.exist?(File.join(__dir__, "gschemas.compiled"))
+  ENV["GSETTINGS_SCHEMA_DIR"] = __dir__
```

Comment:
Could you add a comment that we must set `GSETTINGS_SCHEMA_DIR` before `require "gtk4"` because `GSETTINGS_SCHEMA_DIR` is used in GIO initialization?